drivers/char/hw_random/bcm2835_rng_rust.rs

```diff
@@ -19,6 +20,20 @@ module! {
     license: b"GPL v2",
 }
 
+struct RngDriver;
+
+impl PlatformDriver for RngDriver {
+    fn probe(device_id: i32) -> Result {
```

Comment:
Is it correct to assume that we'll eventually provide more information than `device_id`?

Reply:
Absolutely!! `device_id` is just a placeholder. Watch this space :)